site/posts/numpy-faq.ipynb

```diff
@@ -5,8 +5,57 @@
    "id": "printable-testing",
    "metadata": {},
    "source": [
-    "Write your post here."
+    "Here is a compilation of questions and issues that arose during the [Numpy](https://colab.research.google.com/drive/1zcjaGXofLzUXkQzJKnANE9Z3yFUt0q7E?usp=sharing) session of the Python Tutorial Seminar Series. \n",
+    "\n",
+    "The live video recording of this content can be found [here](https://www.youtube.com/watch?v=kstc-6uz7AQ&t=10s).\n",
+    "\n",
+    "------"
    ]
+  },
+  {
+   "source": [
+    "**Q.** Is it possible to provide a “step” instead of a number of elements when using `numpy.linspace` to create an array?\n",
+    "\n",
+    "**A.** The step size is derived from the input arguments `start`, `stop`, and `number` with `numpy.linspace`. If you instead want to supply the step size and derive the number of steps, you would instead use the function `numpy.arange` that takes the input arguments `start`, `stop`, and `number`. `numpy.arange()` differs from Python’s built-in `range()` function which only supports integers and produces a `list` rather than an `array`.\n",
+    "\n",
+    "-----\n",
+    "<!-- TEASER_END -->"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Are there Python methods that mutate the object, or does they always generate a new object? If so, are there any good heuristics for knowing which methods change their objects and which generate changed copies? \n",
+    "\n",
+    "**A.** Yes there are Python functions that mutate the object. The inplace operations are usually defined at the library/package level. Checking the documentation is the safest way to confirm how the library or package behaves.\n",
+    "\n",
+    "----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Is it faster to work with a Python `list` or a Numpy `array`?\n",
+    "\n",
+    "**A.** Arrays are faster if you want to modify the object. You can test which is faster in a Jupyter by typing `%%time` at the top of your cells, or %%timeit which will run the cell multiple times and produce the average time of computation.\n",
+    "\n",
+    "-----"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "source": [
+    "**Q.** Can numpy Arrays hold non-numeric data?\n",
+    "\n",
+    "**A.** Yes, a Numpy `array` can hold any data type of equivalent size as well as any Numpy native data type. It cannot hold varying like `string`s, but `string`s of the same lengths are acceptable.\n",
```

Comment:
It should also probably be stated (if it wasn't obvious from the tutorial) that arrays hold data of the *same type*.  That is, the type of every element in the array must be the same.  If that wasn't clearly stated in the tutorial, then if should be stated here.  (And maybe just for clarity, it should be stated here again.)

Reply:
added a line on this!